components/WalletProvider.tsx

```diff
@@ -16,6 +16,7 @@ const { chains, provider } = configureChains(
 )
 
 const { connectors } = getDefaultWallets({
+  // Change appName to D_D Academy
```

Comment:
Should we make this change now?

Reply:
I left it considering we aren't using wallet connection just yet. Totally fine with changing it now!

Reply:
Ah cool, either way works! Dealer's (Author's) choice! ☀️

Reply:
Changed it.